src/coreclr/vm/ilstubresolver.h

```diff
@@ -108,6 +108,24 @@ class ILStubResolver : public DynamicResolver
     PTR_LoaderHeap          m_loaderHeap;
 };
 
+class AsyncResumeILStubResolver : public ILStubResolver
```

Comment:
Do we still need this with the latest scheme?

Reply:
No, this can be removed again